docs/ci-services.md

```diff
@@ -0,0 +1,55 @@
+# CI Services
+
+The CI service blob defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  TravisCI: {scaffolder: travisScaffolder, public: true}
+}
+```
+
+## Config
+
+### keys: __string__
+
+Keys in the CI services blob define the options the user will be presented with
+when choosing their desired CI service.
+
+example: `TravisCI`
+
+### values: __object__
+
+Values in the CI services blob define the actual mechanisms and configuration
+that will be used by the JavaScript Scaffolder to scaffold the CI service for
+the user's project.
+
+#### scaffolder: __function__ (_required_)
+
+Consumer-defined plugin(??) called when this ci service is selected by the
+user in the CLI. Will be called with the following arguments:
```

Comment:
```suggestion
user in the interactive prompt. Will be called with the following arguments:
```

Reply:
i think this is just a little less specific about the consumer of the scaffolder but still speaks to the input needed that is specific to the scaffolder